testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_concurrency.py

```diff
@@ -0,0 +1,209 @@
+import pytest
+
+from keywords.MobileRestClient import MobileRestClient
+from keywords import document
+from CBLClient.Replication import Replication
+from CBLClient.Authenticator import Authenticator
+from libraries.testkit import cluster
+from CBLClient.Document import Document
+
+
+def create_doc_from_document(doc_id, content, channel):
```

Comment:
This is similar to keywords/document::create_doc. Why can't we reuse existing code?

Reply:
looks like i had some issue while using directly. With latest code, I reused the code document:create_doc and tested on cbl20-ver1 and worked fine. I modified this PR to use existing code